src/features/Settings/components/AddressBook.tsx

```diff
@@ -250,17 +245,30 @@ export default function AddressBook({
             <SEditableText
               truncate={true}
               value={label}
-              saveValue={(value) => updateContact({ address, label: value, network, notes, uuid })}
+              onChange={(value) => updateContact({ address, label: value, network, notes, uuid })}
             />
           </Label>,
           <EthAddress address={address} truncate={true} isCopyable={true} />,
           <Network color={color}>{network}</Network>,
           <EditableText
+            placeholder="(empty)"
             truncate={true}
             value={notes}
-            saveValue={(value) => updateContact({ address, label, network, notes: value, uuid })}
+            onChange={(value) => updateContact({ address, label, network, notes: value, uuid })}
           />,
-          <DeleteButton onClick={() => setDeletingIndex(index)} icon="exit" />
+          <>
+            {isMobile ? (
+              <Box key={index}>
+                <LinkApp href="#" onClick={() => setDeletingIndex(index)}>
+                  {translateRaw('ADDRESSBOOK_REMOVE')}
```

Comment:
It works in Balances because the text size gives the illusion that it is centred.
Removing the header in this table makes it obvious that there is a miss alignment.
Choosing not.